.github/workflows/drizzle_ci.yml

```diff
@@ -82,3 +82,27 @@ jobs:
     - name: Run flake8
       run: |
         flake8 drizzle --count --show-source --statistics
+
+  code_coverage:
+    name: Code Coverage Report
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -e .[test]
+        python -m pip install pytest-cov
```

Comment:
Again, `pytest-cov` should go into the `[test]` extras dependencies in setup.py so it doesn't have to be installed here.

Reply:
I made this change.